### PR TITLE
worked on fixes

### PR DESCRIPTION
**Pull request by Stevenkwizera06**
What does this PR do?

Designing this figma website https://www.figma.com/file/f07DCCsOluL9q0HsgSXAJM/FREEBIES-Landingpage-LaslesVPN-(Community)?node-id=1%3A2

Description of Task to be completed?

-I designed using only tailwind  

How should this be manually tested?

Clone the repo

checkout to branch Dev

Run npm install to install modules

Run npm run start

* Here is the desktop view
Screen Shot 2022-10-15 at 12 08 12

* Here is the mobile view
Screen Shot 2022-10-15 at 13 09 07
Screen Shot 2022-10-15 at 13 09 31

### CODE REVIEW

**Review by IbrahimBagalwa — COMMENTED** [review]
## Quality Assurance Wrong
It looks good, great job :)
For quality assurance you should first follow the figma file [here](https://www.figma.com/file/f07DCCsOluL9q0HsgSXAJM/FREEBIES-Landingpage-LaslesVPN-(Community)?node-id=0%3A1)
and then reproduce exactly the same thing.
eg: - font-size, font-family, color, background-color, ...

File: index.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 <body class="mx-auto lg:body__class-container bg-[#F6F6F6]">
```
Comment (chrisostome01) [inline index.html]: Let's now sticky to tailwindcss only we can do BEM on the project

File: index.html [hunk 2]
```diff
@@ -12,7 +12,7 @@
...
 
-    <header class="fixed top-0 bg-white-500 w-full shadow-lg">
+    <header class="fixed top-0 bg-white-500 w-full shadow-lg z-40">
         <nav class="text-black bg-white flex justify-between ml-0 p-4">
             <div class="flex items-center ">
                 <svg class="h-8 w-auto " width="149" height="37" viewBox="0 0 149 37" fill="none"
```
Comment (chrisostome01) [inline index.html:32]: Try to make file for this svg

**Review by chrisostome01 — CHANGES_REQUESTED** [review]
Well done @Stevenkwizera06, i left some comments for you may check them out please 
You can import the `Rubik`  font family as the design says and also header is not that stick and the shadow is not necessary at all

**Pull request merged**